app/models.py

```diff
@@ -352,6 +352,21 @@ class OrganisationTypes(db.Model):
     annual_free_sms_fragment_limit = db.Column(db.BigInteger, nullable=False)
 
 
+class OrganisationPermission(db.Model):
+    __tablename__ = "organisation_permissions"
+    id = db.Column(UUID(as_uuid=True), primary_key=True, nullable=False, default=uuid.uuid4)
+
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    organisation = db.relationship("Organisation")
+    organisation_id = db.Column(UUID(as_uuid=True), db.ForeignKey("organisation.id"), nullable=False)
+    organisation_permission = db.Column(
+        db.Enum(*ORGANISATION_PERMISSION_LIST, name="organisation_permission_types"),
```

Comment:
Could we just call this `permission`? I think that matches what we have on other models, and otherwise we have `OrganisationPermission` as the table name as well as `organisation_permission` as a column name - feels a bit verbose/redundant.

Reply:
Sure.